homeassistant/components/light/hue.py

```diff
@@ -54,6 +54,10 @@ def _find_host_from_config(hass, filename=PHUE_CONFIG_FILE):
 
 def setup_platform(hass, config, add_devices_callback, discovery_info=None):
     """ Gets the Hue lights. """
+    if config.get('ignore_unreachable', False):
+        _CONFIGURING['ignore_unreachable'] = True
```

Comment:
That's a good point about that dict, I didn't think about using a local. I wanted something that would persist in case new link bulbs were added while the ha was running. 

I can change that other too while I'm updating the `_CONFIGURING` variable replacement.